LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2013-2015
```

Comment:
You can just wrote "Many open-source contributors" Offended persons can add their real name and dates.